core/ir/aarch64/instr.c

```diff
@@ -35,18 +35,23 @@
 #include "instr.h"
```

Comment:
> Lots of tests are currently failing. Probably due to the change in instr_t size.

There is a "black box" definition with the size.  See e.g. #6237 which is the last change to the struct.  If there's no padding slot we can hide in and the size has changed for 64-bit, we should bump the minor version to 90 as well (see https://dynamorio.org/page_new_release.html#autotoc_md230).

Reply:
There is a byte slot next to the 3 rep_rel_pos,num_{dsts,srcs}: but that would limit us to 8 bits of ISA flags (currently using 6 with synth addition?).  This would not change either 32-bit or 64-bit sizes if I'm looking at it right.  Hmm, worth considering?

Reply:
I went with your suggestion (thank you!).
Also, dr_isa_mode_t doesn't need to be a "flag", so with 1 byte we still have plenty of ISA modes.

Reply:
Ah, right, it's a regular enum, so 1 byte should be plenty as you said.